deployments/k8s/helm/signalfx-agent/templates/_helpers.tpl

```diff
@@ -55,6 +55,13 @@ Create the name of the secret holding the token
 {{- define "signalfx-agent.secretName" -}}
     {{ default (include "signalfx-agent.fullname" .) .Values.signalFxAccessTokenSecretName }}
 {{- end -}}
+{{/*
+Name of secret holding splunk hec token
+*/}}
+{{- define "signalfx-agent.secretSplunkName" -}}
```

Comment:
Call this `splunkSecretName` or `secretNameSplunk` instead of sticking `splunk` in the middle.

Reply:
Modified var name